paragon/_overrides.scss

```diff
@@ -17,75 +17,34 @@ header {
     // it breaks the UI.
     height: calc(3.75rem + #{$header-border-top-width + $header-border-bottom-width});
   }
-  // Set the color of text in the header - course about page.
-  @if variable-exists(main-nav-color) {
-    span {
-      color: $main-nav-color;
-    }
-  }
-
-  .nav-links {
-    @if variable-exists(main-nav-link-color) {
-      // Set the color of the links in the dashboard
-      .main .nav-tab a,
-      .secondary .nav-item a,
-      .secondary .nav-item a span,
-      .toggle-user-dropdown span {
-        color: $main-nav-link-color !important;
-      }
-    }
-
-    @if variable-exists(main-nav-item-border-bottom-color) {
-      // Set the bottom border color of the links in the dashboard like 'Courses', 'Programs', 'Profile' etc.
-      .main .nav-tab a {
-        &.active {
-          border-bottom-color: $main-nav-item-border-bottom-color !important;
-          border-bottom-style: solid !important;
-        }
-      }
-    }
-
-    @if variable-exists(main-nav-item-hover-border-bottom-color) {
-      .main .nav-tab a {
-        &:not(.active):hover {
-          border-bottom-color: $main-nav-item-hover-border-bottom-color !important;
-          border-bottom-style: solid !important;
-        }
-      }
-    }
 
-    @if variable-exists(user-dropdown-color) {
-      // Customize the color of the links in the user dropdown menu
-      nav.secondary-menu-container .dropdown-menu .dropdown-item a {
+  @if variable-exists(user-dropdown-color) {
+    // Customize the color of the links in the user dropdown menu in the learning MFE
+    .user-dropdown,
+    // Customize the color of the links in the user dropdown menu in other MFEs
+    nav.secondary-menu-container {
+      .dropdown-menu a.dropdown-item {
```

Comment:
This header seems to be inconsistent between MFEs. I didn't dig into this, though.